public/index.html

```diff
@@ -37,4 +37,5 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
   </body>
+
```

Comment:
Nit: this change is unnecessary. Please consider reverting if not too much trouble.